src/command.ts

```diff
@@ -186,4 +222,12 @@ export default abstract class Command {
       console.error(error)
     }
   }
+
+  protected toSuccessJson(result: Record<string, any>): Record<string, any> {
+    return {result}
```

Comment:
Just return result? People transitioning from `sfdx` -> `sf` would need to remove the `.result` but I think that is cleaner.